src/shortcuts/archive.tsx

```diff
@@ -73,7 +73,8 @@ const archiveShortcut: Shortcut = {
   gesture: 'ldl',
   svg: Icon,
   keyboard: { key: Key.Backspace, shift: true, meta: true },
-  canExecute: getState => isDocumentEditable() && !!getState().cursor,
+  isActive: getState => isShortcutExecutable(getState),
+  canExecute: getState => isShortcutExecutable(getState),
```

Comment:
Instead of copying `canExecute` to `isActive`, change the Toolbar logic to fallback to `canExecute` when there is no `isActive`.

https://github.com/cybersemics/em/blob/862aa3b7e97f5904999b1abb9fee3640880d5d74/src/components/Toolbar.tsx#L215

That way there is less duplication and fewer required fields for new shortcuts.